### PR TITLE
Update PyCharm (soon → beta)

### PR DESCRIPTION
**Pull request by corneliusludmann**
## Description
This PR updated the docs page for PyCharm IDE.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/7022

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1297"></a>

### CODE REVIEW

**Comment by corneliusludmann** [comment]
@loujaybee Is that what we need on the docs side for PyCharm?

**Comment by loujaybee** [comment]
Could we also add PyCharm into this list: 

image

On: https://www.gitpod.io/docs/editors

**Comment by loujaybee** [comment]
Each of the pages actually mentions IntelliJ / GoLand being in Beta. 

> We currently have support for IntelliJ and GoLand in Beta

Could we do either:

A) Update this text to include PyCharm
B) Remove / make the copy text more agnostic so we don't have to keep re-updating?

image

File: src/routes/docs/editors/goland.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ title: GoLand
 
 ## Desktop
 
-We currently have support for JetBrains GoLand (in private Beta). If you're interested to be part of the private Beta group, [leave a comment in this GitHub issue](https://github.com/gitpod-io/gitpod/issues/6342). If you have any issues with the IntelliJ GoLand Beta, you can [leave feedback in this GitHub issue](https://github.com/gitpod-io/gitpod/issues/6576).
```
Comment (loujaybee) [inline src/routes/docs/editors/goland.md:14]: Ooops! Great catch !

File: src/routes/docs/editors/intellij.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ title: IntelliJ IDEA
 
 ## Desktop
 
-We currently have support for JetBrains IntelliJ IDEA (in private Beta). If you're interested to be part of the private Beta group, [leave a comment in this GitHub issue](https://github.com/gitpod-io/gitpod/issues/6342). If you have any issues with the IntelliJ GoLand Beta, you can [leave feedback in this GitHub issue](https://github.com/gitpod-io/gitpod/issues/6576).
```
Comment (loujaybee) [inline src/routes/docs/editors/intellij.md:14]: Also, great catch.

**Review by loujaybee — APPROVED** [review]
Thanks for picking this one up @corneliusludmann !

Added a couple of notes on some other pages to update, otherwise, it looks good to me, I'll approve early, if we address the above comments, then let's get it shipped 🚢  🚢. 🚢  !

**Pull request merged**